package.json

```diff
@@ -98,30 +98,30 @@
     "html5parser": "^2.0.2",
     "jsdom": "^20.0.0",
     "kolorist": "^1.5.1",
-    "p-queue": "6.6.2",
+    "p-queue": "7.3.0",
```

Comment:
Can you avoid bumping other deps? (we could do it in another PR)

Reply:
Sorry, I didn't notice this, and it has been revised.